src/Containers/DropArea/DropArea.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import styled from 'styled-components';
+import { MainInterface, Main } from '@Utils/BaseStyles';
+import { flex } from '@Utils/Mixins';
+import Dropzone, {
+    useDropzone,
+    DropEvent,
+    FileRejection,
+} from 'react-dropzone';
+
+export interface IDropAreaProps
+    extends MainInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    message?: string;
+    onClick?: React.MouseEventHandler<HTMLElement>|undefined;
+    onDragEnter?: React.DragEventHandler<HTMLElement> | undefined;
+    onDragLeave?: React.DragEventHandler<HTMLElement> | undefined;
+    onDropHandler?:
+        | (<T extends File>(
+              acceptedFiles: T[],
+              fileRejections: FileRejection[],
+              event: DropEvent,
+          ) => void)
+        | undefined;
+    isDragEnter?: boolean;
+}
+
+export const DropArea: React.FC<IDropAreaProps> = ({
+    message = 'Drag & Drop your files here',
+    onClick=()=>null,
+    onDragEnter = () => null,
+    onDragLeave = () => null,
+    onDropHandler = () => null,
+    isDragEnter = false,
+    ...props
+}): React.ReactElement => {
+    const { getInputProps, getRootProps } = useDropzone({
+        onDragEnter,
+        onDragLeave,
+        onDrop: onDropHandler,
+    });
+    return (
+        <Dropzone multiple>
+            {() => (
+                <div
+                    {...getRootProps({
+                        onClick: (e: React.MouseEvent) => {
+                            e.stopPropagation();
+                        },
```

Comment:
Do you mean CSS pointer events none??

Reply:
I don't know what you mean. this is to restrict click behaviour to the button.

Reply:
Google Pointer Events CSS "Pass Through :)